### ISSUE TITLE
Please bring to 2.0.  Still better than Factoriopedia!

### ISSUE THREAD

**Issue by Identifier** [raiguard/RecipeBook#170]
Please update this awesome mod to factorio 2.0.  Factoriopedia is nice, but your mod does so much more!  This mod an Factory Planner are must haves making the factory grow!

**Comment by raiguard** [comment]
The mod as you know it will not be updated, but I am considering releasing my FNEI-style experiment just so that someone doesn't go and perform even more necromancy on FNEI. That mod needs to die and stay dead.

**Comment by LOMS** [comment]
> Please update this awesome mod to factorio 2.0. Factoriopedia is nice, but your mod does so much more! This mod an Factory Planner are must haves making the factory grow!

I come here just to post the same message. @raiguard your recipe-book mod is superior.

**Comment by raiguard** [comment]
The 3.0 version of the mod is dead because I no longer wish to maintain this code base. If there are things that you want to have added/changed in factoriopedia, please post on the factorio forums requesting changes. Improving the factoriopedia will benefit all players.

**Comment by stucker9** [comment]
> The mod as you know it will not be updated, but I am considering releasing my FNEI-style experiment just so that someone doesn't go and perform even more necromancy on FNEI. That mod needs to die and stay dead.

I am not involved in the community at all so I'm not aware of any circumstances. It's disappointing to hear it won't be updated but can also understand your position and desire to not update it anymore for whatever reason. Well, thanks for the mod while it lasted lol it was better than anything else.

**Comment by gparker42** [comment]
@raiguard, in case someone else wants to pick this up, do you happen to know what 2.0 incompatibilities are likely to need fixes?

(I might do it myself. I don't think Factoriopedia's design will scale to Py's item and recipe count.)

**Comment by kubiix** [comment]
@raiguard - I'd like to take over maintenance and development if you'd agree.

**Comment by raiguard** [comment]
It has become clear that Factoriopedia is not capable as a recipe book replacement, so I will be updating RB in some capacity. But it won't be the same as the mod you have used.

**Comment by raiguard** [comment]
To set some expectations, v4.0 will not be like previous versions - it will essentially be a modern version of FNEI. v3.x had far too much information overload and most of that is covered in Factoriopedia. RB will rebrand to "a simple recipe browser".

![image](https://github.com/user-attachments/assets/833daa86-57e3-4b83-9ea9-0c9066f72f34)

**Comment by protoplm** [comment]
Recipe Book in its current version is excellent for me - I _like_ the detail and especially the modular interface. I don't understand how people use FNEI. If there's 4 different ways to make an item I'd like to see all 4 on the same page. FNEI requires digging through multiple pages just to get a sense of this. Factoriopedia does this reasonably well but it's too much space and doesn't scale well.

One point of Recipe Book that could use improvement is the distinction between a 'Recipe', an 'Item', and an 'Entity' - searching for an 'Electric Furnace' gave results for all 3 of these and requires some fiddling to get into the page you're actually looking for. If you want to simplify - that can be streamlined. I like the above image for the view of a recipe but can we retain a page for 'Item' that keeps info regarding stack size, links to each of the recipes to make it (color coded on if you can make it with current research, hover over to show what's involved? 

![image](https://github.com/user-attachments/assets/b1e04622-2bb4-43bb-bf46-da94a10a5bba)

This sort of thing would be a mess on Factoriopedia due to the sheer amount of possible uses and the attempt to use a grid of icons rather than a list of names. Clicking on one of these recipes would bring you to the page you showed above.

**Comment by ShadowReaper420** [comment]
I agree with the above post the UI in v3 is so much better than the FNEI style especially in bigger mods like Py, I get the codebase for the old version is a mess but like if I wanted FNEI, I would use FNEI.

**Comment by raiguard** [comment]
I also have a branch I've been sitting on for over two years which is basically Factoriopedia but better. I will release this version.

It will not have feature parity at first but I can make it better over time.

![Image](https://github.com/user-attachments/assets/410ffcad-ebac-45f2-820c-5f1c38470034)

**Comment by raiguard** [comment]
RB4 will be released tomorrow.
